lib/index.js

```diff
@@ -82,18 +86,24 @@ Optimizely.prototype.initialize = function() {
  */
 
 Optimizely.prototype.track = function(track) {
-  var payload = {};
+  // Use the new-style API (which is compatible with Classic and X)
+  var payload = {
+    type: 'event',
+    eventName: track.event(),
+    tags: track.properties()
+  };
   var revenue = track.revenue();
```

Comment:
So if revenue is non numeric, will it be included in `track.properties()`?

I.e., do we need to delete it if `track.revenue()` is falsey?

```
if (revenue) {
  payload.tags.revenue *= 100;
} else {
  remove(payload.tags, 'revenue');
}
```